components/director/internal/domain/eventapi/repository.go

```diff
@@ -1,7 +1,10 @@
 package eventapi
```

Comment:
Please add checks to functions taking `item` as an input (you can see document repository for a reference).

Check also API domain repository.

Reply:
done